### PR TITLE
feat (CI): Enable 1JS VR tool to run on v7

### PR DESCRIPTION
**Pull request by TristanWatanabe**
## Changes:
- bumps agent host machine `node` version to version `14.18.1` to align with master and since it's incompatible with the `storywright` tool.
- cherry-picks yaml files added for master branch slightly modified to run for the v7 branch instead.
- adds `convert`, `build-storybook`, `vrt:build` scripts needed to run vr tests in pipeline.
- adds necessary 1JS vr-tool dev dependencies: `storywright` and `vrscreenshotdiff`

**Note**: The v7 branch will share the `VR Tool Pipeline - PR` pipeline with the master branch but will have its own separate `VR Tool - Baseline (v7)` pipeline.

## Related Issue(s)

* Part of #25078

### CODE REVIEW

File: .devops/templates/tools.yml [hunk 1]
```diff
@@ -2,7 +2,8 @@
 steps:
   - task: NodeTool@0
     inputs:
-      versionSpec: '12.x'
+      versionSpec: '14.18.1'
```
Comment (TristanWatanabe) [inline .devops/templates/tools.yml:5]: bumps agent host machine `node` version to version `14.18.1` to align with master (#25946) and to make it compatible with the `storywright` tool.

**Comment by fabricteam** [comment]
## [Perf Analysis](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">7.0 Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/BaseButton-mount_base.html">767</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/BaseButton-mount.html">772</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/Breadcrumb-mount_base.html">34431</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/Breadcrumb-mount.html">34694</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/Checkbox-mount_base.html">1267</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/Checkbox-mount.html">1270</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/CheckboxBase-mount_base.html">1103</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/CheckboxBase-mount.html">1073</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/ChoiceGroup-mount_base.html">4080</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/ChoiceGroup-mount.html">4037</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/ComboBox-mount_base.html">678</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/ComboBox-mount.html">741</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/CommandBar-mount_base.html">6564</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/CommandBar-mount.html">6499</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/ContextualMenu-mount_base.html">10525</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/ContextualMenu-mount.html">10516</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/DefaultButton-mount_base.html">929</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/DefaultButton-mount.html">946</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/DetailsRow-mount_base.html">2982</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/DetailsRow-mount.html">2957</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/DetailsRowFast-mount_base.html">3039</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/DetailsRowFast-mount.html">2977</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/DetailsRowNoStyles-mount_base.html">2835</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/DetailsRowNoStyles-mount.html">2863</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/Dialog-mount_base.html">1779</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/Dialog-mount.html">1785</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/DocumentCardTitle-mount_base.html">1520</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/DocumentCardTitle-mount.html">1524</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/Dropdown-mount_base.html">2005</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/Dropdown-mount.html">2030</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/FocusTrapZone-mount_base.html">1437</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/FocusTrapZone-mount.html">1412</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/FocusZone-mount_base.html">1496</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/FocusZone-mount.html">1505</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/IconButton-mount_base.html">1380</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/IconButton-mount.html">1377</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/Label-mount_base.html">307</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/Label-mount.html">307</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/Layer-mount_base.html">1637</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/Layer-mount.html">1627</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/Link-mount_base.html">404</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/Link-mount.html">404</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/MenuButton-mount_base.html">1176</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/MenuButton-mount.html">1185</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/MessageBar-mount_base.html">1758</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/MessageBar-mount.html">1772</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/Nav-mount_base.html">2591</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/Nav-mount.html">2630</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/OverflowSet-mount_base.html">1229</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/OverflowSet-mount.html">1218</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/Panel-mount_base.html">1197</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/Panel-mount.html">1155</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/Persona-mount_base.html">801</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/Persona-mount.html">803</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/Pivot-mount_base.html">1120</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/Pivot-mount.html">1125</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/PrimaryButton-mount_base.html">1052</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/PrimaryButton-mount.html">1052</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/Rating-mount_base.html">6711</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/Rating-mount.html">6657</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/SearchBox-mount_base.html">1074</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/SearchBox-mount.html">1059</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/Shimmer-mount_base.html">2252</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/Shimmer-mount.html">2243</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/Slider-mount_base.html">1300</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/Slider-mount.html">1299</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/SpinButton-mount_base.html">3919</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/SpinButton-mount.html">3931</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/Spinner-mount_base.html">385</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/Spinner-mount.html">382</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/SplitButton-mount_base.html">2410</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/SplitButton-mount.html">2444</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/Stack-mount_base.html">454</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/Stack-mount.html">451</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/StackWithIntrinsicChildren-mount_base.html">1389</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/StackWithIntrinsicChildren-mount.html">1380</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/StackWithTextChildren-mount_base.html">4291</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/StackWithTextChildren-mount.html">4287</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/SwatchColorPicker-mount_base.html">8060</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/SwatchColorPicker-mount.html">8207</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/TagPicker-mount_base.html">2162</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/TagPicker-mount.html">2138</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/TeachingBubble-mount_base.html">41212</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/TeachingBubble-mount.html">41224</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/Text-mount_base.html">404</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/Text-mount.html">399</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/TextField-mount_base.html">1135</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/TextField-mount.html">1142</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/Toggle-mount_base.html">736</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/Toggle-mount.html">736</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/button-mount_base.html">88</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25946/perf-test/button-mount.html">91</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Review by micahgodbolt — APPROVED** [review]

File: apps/vr-tests/convertStoriesToStoryWright.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+#!/usr/bin/env bash
+
+find ./src -type f |while read fname; do
+  impor=""
```
Comment (GeoffCoxMSFT) [inline apps/vr-tests/convertStoriesToStoryWright.sh:6]: import="" ?

Comment (GeoffCoxMSFT) [inline apps/vr-tests/convertStoriesToStoryWright.sh:6]: impor is a strange name

**Review by GeoffCoxMSFT — APPROVED** [review]

File: apps/vr-tests/convertStoriesToStoryWright.sh (hunk 2 above)
Comment (TristanWatanabe) [reply apps/vr-tests/convertStoriesToStoryWright.sh:6]: Will address in follow-up PRs with cleanups!

**Pull request merged**